### PR TITLE
AB2D-1340: AB2D client certificate for BFD Production

### PR DESCRIPTION
**Pull request by dkrylovsb**
This PR adds the necessary certificates for the AB2D Prod environment to connect to the BFD Prod environment. Self signed certs are generated and linked via the env_specific.yml file in the playbooks-ccs directory. Followed the pattern established in other PRs along these lines.

### CODE REVIEW

**Comment by njdister** [comment]
@dkrylovsb Unfortunately you may have to open this from our repo rather than a fork, as two of our CI tests necessary to merge require credentials from our repo.

File: ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+mMDnsEdkXtQBSwpd/u7r3qaHR/RvibJMjAOJW2iGhsc2776/nJDuzyrX8RZn4wTo
+ADhaz8Y5JHcEznHSVYvEoEXGXx4xXv8U6yQZmk2x0xzQ5fB674/uuZFWfi7GMnco
+bC04DyCJUA07Rupgk6nCoSVdoOS0zSZ5RpYSEmokkABeqH3UcDbVmgRkhjFXFTLk
+Y4nUJknE+L4OkmggI2MyIa0Dn1m0mSx3BFC5r2LvyXiYIMxJeuxD+ohZtCH++0QB
+qyRWAPaKLkNTnhJgefp5kC95l3IFr+kfAi+ub3M0V8laarprucaXUgIpAFjHlb65
+/AG92G4FQx4NrWt511HiDqYdNzt+0eJ8sSIk5AuZNkLq2XFtP8j3rTOzT3edu+Ih
+vejQJ8SmpO7mJOo320q1hBEI4LsUfdbAufQQWd2Ti9lnjfIO4qwFsAdnBTBGbKLY
+GkJFhMc0aQjhUhsaG5v7bv5V55IPc0ns/NeroI1Cr4UStKbMYlw89ztLLVF2cPtt
+MXIEdz5t
+-----END CERTIFICATE-----
```
Comment (RickHawesUSDS) [inline ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem:28]: I'm not able to dump this certificate by hand. 
```
openssl x509 -in ~/projects/temp/AB2D.pem  -text -noout                                                                                                                                                 09:26:27
unable to load certificate
4604417644:error:09FFF066:PEM routines:CRYPTO_internal:bad end line:/BuildRoot/Library/Caches/com.apple.xbs/Sources/libressl/libressl-22.260.1/libressl-2.6/crypto/pem/pem_lib.c:791:
```

Just to double check my command line, I ran it against a known good cert. 
```
openssl x509 -in ~/projects/beneficiary-fhir-data/ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem  -text -noout                                                             09:26:28
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 17737621763724056112 (0xf628b10423029630)
    Signature Algorithm: sha256WithRSAEncryption
```

Comment (dkrylovsb) [reply ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem:28]: Hi @RickHawesUSDS :
`bash-3.2$ curl -o ~/Temp/AB2D.pem https://raw.githubusercontent.com/CMSgov/beneficiary-fhir-data/f3df72b64ef1ec6a8cdbb48c16ffe4ac28557fde/ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem; openssl x509 -in ~/Temp/AB2D.pem  -text -noout`

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1688  100  1688    0     0   3059      0 --:--:-- --:--:-- --:--:--  3063
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 9718410767053974825 (0x86debb1a2f956d29)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-prod-client
```

🤷‍♂️

Comment (jzulim) [reply ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem:28]: @RickHawesUSDS and @dkrylovsb Cert looks good on my end. 

```jzmbp:tmp johnzulim$ openssl x509 -in client_data_server_ab2d_prod_certificate.pem -text -noout```
```Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number:
            86:de:bb:1a:2f:95:6d:29
        Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN = ab2d-prod-client
        Validity
            Not Before: Apr 13 18:29:05 2020 GMT
            Not After : Apr 13 18:29:05 2021 GMT
        Subject: CN = ab2d-prod-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                RSA Public-Key: (4096 bit)
                Modulus:```

**Comment by jzulim** [comment]
@dkrylovsb as @njdister pointed out, you will need to open this from our repo rather than a fork, as two of our CI tests necessary to merge require credentials from our repo.

**Comment by dkrylovsb** [comment]
@jzulim Nobody from our team has permissions to submit PRs against this repo 😬

**Comment by njdister** [comment]
Closing for #259

**Pull request closed without merging**